lib/sidekiq.rb

```diff
@@ -149,7 +149,12 @@ def self.default_server_middleware
   end
 
   def self.default_worker_options=(hash)
-    @default_worker_options = default_worker_options.merge(hash.stringify_keys)
+    h = hash.dup
+    # stringify
+    h.keys.each do |key|
+      h[key.to_s] = h.delete(key)
+    end
+    @default_worker_options = default_worker_options.merge(h)
```

Comment:
That's fine (I find it more readable but if you find it less readable this is a good enough argument to me). I just think that this kind of micro-optimization doesn't make much sense in this case. How many possible keys we're talking about after all? ;) It will happen just once :)